### ISSUE TITLE
Database migration fails (v2.4.4 -> v.2.5.2)

### ISSUE THREAD

**Issue by iDemonix** [digitalocean/netbox#2741]
The upgrade.sh script is barfing after installing all of the package updates. Pip satisfies all requirements, but then database migration and 'collecting static files' fails with a syntax error.

Using Python 3.4.5.

```
Applying database migrations (python3 netbox/manage.py migrate)...
Traceback (most recent call last):
  File "netbox/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
  File "/usr/lib/python3.4/site-packages/django/core/management/__init__.py", line 11, in <module>
    from django.conf import settings
  File "/usr/lib/python3.4/site-packages/django/conf/__init__.py", line 18, in <module>
    from django.utils.functional import LazyObject, empty
  File "/usr/lib/python3.4/site-packages/django/utils/functional.py", line 12
    return _curried_func(*args, *moreargs, **{**kwargs, **morekwargs})
                                ^
SyntaxError: invalid syntax
Collecting static files (python3 netbox/manage.py collectstatic --no-input)...
Traceback (most recent call last):
  File "netbox/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
  File "/usr/lib/python3.4/site-packages/django/core/management/__init__.py", line 11, in <module>
    from django.conf import settings
  File "/usr/lib/python3.4/site-packages/django/conf/__init__.py", line 18, in <module>
    from django.utils.functional import LazyObject, empty
  File "/usr/lib/python3.4/site-packages/django/utils/functional.py", line 12
    return _curried_func(*args, *moreargs, **{**kwargs, **morekwargs})
                                ^
SyntaxError: invalid syntax

```

**Comment by iDemonix** [comment]
I've just realised the requirement was Python 3.5, so that's more than likely my issue...

**Comment by iDemonix** [comment]
Solved by upgrading to Python 3.6 and actually reading the instructions 😄
